var/spack/repos/builtin/packages/openmpi/package.py

```diff
@@ -237,6 +237,7 @@ class Openmpi(AutotoolsPackage):
             description='Enable MPI_THREAD_MULTIPLE support')
     variant('cuda', default=False, description='Enable CUDA support')
     variant('pmi', default=False, description='Enable PMI support')
+    variant('cxx', default=False, description='Enable C++ MPI bindings')
     variant('cxx_exceptions', default=True, description='Enable C++ Exception support')
```

Comment:
These two options seem related (i.e. I would expect C++ exception support is there only if C++ bindings are there). Can you double check that and eventually merge the two into a multi-valued variant?

Reply:
these are actually different things.  I only see one problem with adding the c++ bindings variant, namely they will be removed in the next major OMPI release stream - v5.0.x.  I'd suggest qualifying the use of -enable-mpi-cxx since the last release stream where the c++ bindings are supported is 4.0.x.

Reply:
in that light may as well qualify the ```--enable-cxx-exceptions``` config arg since it will also be moved to a ```deleted``` config options list in 5.0.0 (already the case in master).